sdk/videoanalyzer/azure-media-videoanalyzer-edge/samples/sample_lva.py

```diff
@@ -6,8 +6,8 @@
 from azure.iot.hub.models import CloudToDeviceMethod, CloudToDeviceMethodResult
 from datetime import time
 
-device_id = os.getenv("iothub_deviceid");
-module_d = os.getenv("iothub_moduleid");
+device_id = os.getenv("iothub_deviceid"); # cspell: disable-line
+module_d = os.getenv("iothub_moduleid"); # cspell: disable-line
```

Comment:
Sorry I should have been more clear! Can we move all incline cspell statements in this PR to the cspell.json?

Reply:
I moved all incline cspell statements.